inc/class-core-sitemaps-provider.php

```diff
@@ -215,6 +212,8 @@ public function get_object_sub_types() {
 		 *
 		 * @link https://github.com/GoogleChromeLabs/wp-sitemaps/pull/72#discussion_r347496750
 		 */
-		return array( false );
+		return array(
+			'' => (object) array( 'name' => '' ),
```

Comment:
The docblock above is no longer applicable.

Also, wouldn't an empty array suffice?

Reply:
@swissspidy The comment actually is still applicable to a degree, it's just slightly off in the detail that the value is now not `false`. The doc-block basically states why this should _not_ be an empty array even though that would look like the reasonable thing to return here. There needs to be a value of some sort in there because otherwise sitemap providers without subtypes result in these sitemaps not being registered.

I actually think this is really strange behavior in the way the current code works, and I agree with you that this _should_ normally return an empty array - it just doesn't work here based on what the current code does. I think we should revisit and potentially fix this in a separate issue.

Reply:
I've updated the content of the block a bit to clarify.

Reply:
Makes sense. Could you create such an issue?

Reply:
@felixarntz created #180.